### PR TITLE
V0 edits

### PR DESCRIPTION
**Pull request by pechouc**
Hello!

This PR aims at introducing the following features / edits:

- Simplified alert screen without any frame being displayed;
- Simplified alert acknowledgement workflow with an intermediary confirmation modal;
- A limit to the number of frames being displayed in alert modals;
- The replacement, by the red alert marker, of the blue site marker. 

A few other small edits have been made:

- The number of devices displayed in the popup of each site marker is now variabilized (set to 4 initially);
- Some bugs that remained in relation with the alert workflow have been fixed. 

Many thanks in advance for the review!

### CODE REVIEW

**Review by Akilditu — CHANGES_REQUESTED** [review]
After re-testing there is no issue related to alert disappearance once acknowledged

File: app/alerts.py [hunk 1]
```diff
@@ -144,20 +148,37 @@ def build_sites_markers(dpt_code=None):
...
-        site_name = row['name']
-        markers.append(dl.Marker(id=f'site_{site_id}',    # Necessary to set an id for each marker to receive callbacks
-                                 position=(lat, lon),
-                                 icon=icon,
-                                 children=[dl.Tooltip(site_name),
-                                           dl.Popup([html.H2(f'Site {site_name}'),
-                                                     html.P(f'Coordonnées : ({lat}, {lon})'),
-                                                     html.P(f'Nombre de caméras : {4}')])]))
+        # We do not output a marker if the site is associated with a live alert being displayed
+        if row['name'].replace('_', ' ').title() in sites_with_live_alerts:
```
Comment (Akilditu) [inline app/alerts.py:152]: nice 👍

**Review by Akilditu — APPROVED** [review]
Everything looks super fine to me !

**Comment by frgfm** [comment]
@Akilditu don't forget to delete the branch once a PR is merged ;)

**Pull request merged**